src/lib/gadgets/gadgets.ts

```diff
@@ -62,4 +62,58 @@ const Gadgets = {
   rot(field: Field, bits: number, direction: 'left' | 'right' = 'left') {
     return rot(field, bits, direction);
   },
+
+  /**
+   * Performs a left shift operation on the provided {@link Field} element.
+   * This is akin to the `<<` shift operation in JavaScript, where bits are moved to the left.
+   * The `leftShift` function uses the rotation method internally to achieve this operation.
+   *
+   * **Note:** You cannot shift {@link Field} elements that exceed 64 bits.
+   * For elements that exceed 64 bits, this operation will fail.
+   *
+   * @param field {@link Field} element to shift.
+   * @param bits Amount of bits to shift the {@link Field} element to the left.
+   *
+   * @throws Throws an error if the input value exceeds 64 bits.
+   *
+   * @example
+   * ```ts
+   * const x = Provable.witness(Field, () => Field(12));
+   * const y = leftShift(x, 2); // left shift by 2 bits
+   * y.assertEquals(48);
+   *
+   * const xLarge = Provable.witness(Field, () => Field(12345678901234567890123456789012345678n));
+   * leftShift(xLarge, 32); // throws an error since input exceeds 64 bits
+   * ```
+   */
+  leftShift(field: Field, bits: number) {
+    return leftShift(field, bits);
+  },
+
+  /**
+   * Performs a right shift operation on the provided {@link Field} element.
+   * This is similar to the `>>` shift operation in JavaScript, where bits are moved to the right.
+   * The `rightShift` function utilizes the rotation method internally to implement this operation.
+   *
+   * **Note:** You cannot shift {@link Field} elements that exceed 64 bits.
+   * For elements that exceed 64 bits, this operation will fail.
+   *
+   * @param field {@link Field} element to shift.
+   * @param bits Amount of bits to shift the {@link Field} element to the right.
+   *
+   * @throws Throws an error if the input value exceeds 64 bits.
+   *
+   * @example
```

Comment:
Examples would be clearer if given in binary or hexadecimal format

Reply:
Also addressed in https://github.com/o1-labs/o1js/pull/1194/commits/e74b623cbb9a49699e78973d3511774d1d598eec